packages/connection-manager/src/js/Connection/ConnectionConstants.js

```diff
@@ -0,0 +1,11 @@
+export const CONNECTION_EVENT_OPEN = Symbol("CONNECTION_EVENT_OPEN");
```

Comment:
Let's please split these into state constants and event types and remove the unnecessary prefix as discussed offline and questioned in https://github.com/dcos/dcos-ui/pull/2409#discussion_r138212722